optaplanner-website-root/content/learn/drl-to-constraint-streams-migration.adoc

```diff
@@ -0,0 +1,447 @@
+# Migrating OptaPlanner score DRL to Constraint Streams
+:jbake-description: Migrate deprecated Score DRL to Constraint Streams
+:toc:
+
+Ever since its introduction back in 2020,
+the https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html[Constraint Streams framework] ("CS") has seen great adoption and has been used to implement any problem you can think of.
```

Comment:
- 
- It's not a framework: framework -> API
- active tense => how about we reword the "ever since"
```suggestion
For a couple of years now, the https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html[Constraint Streams API] ("CS") is fully featured and widely adopted.
It can handle any use case written in scoreDRL. It's also faster and https://www.optaplanner.org/blog/2020/04/07/ConstraintStreams.html[easier to use].
```